updating/index.adoc

```diff
@@ -0,0 +1,67 @@
+
+[id="updating-clusters-overview-installing-and-configuring-service"]
+== Installing and configuring the OpenShift Update Service
+xref:../updating/installing-update-service.adoc#installing-update-service[Installing and configuring the OpenShift Update Service]: You can install and configure {product-title} update service by using a hosted service located behind public APIs. However, clusters in a restricted network have no way to access public APIs for update information. For more information, see the following actions:
+
+* xref:../updating/installing-update-service.adoc#update-service-install[Installing the OpenShift Update Service Operator]
+* xref:../updating/installing-update-service.adoc#update-service-create-service[Creating an OpenShift Update Service application]
+* xref:../updating/installing-update-service.adoc#update-service-delete-service[Deleting an OpenShift Update Service application]
+* xref:../updating/installing-update-service.adoc#update-service-uninstall[Uninstalling the OpenShift Update Service Operator]
+
+
+[id="updating-clusters-overview-upgrade-channels-and-releases"]
+== Understanding upgrade channels and releases
+xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels-releases[Understanding upgrade channels and releases]: Upgrade channels allow you to choose an upgrade strategy. Upgrade channels are connected to a minor version of {product-title}. Upgrade channels control only release selection and do not impact the version of the cluster that you install; the openshift-install binary file for a specific version of {product-title} always installs that version. For more information, see the following:
+
+* xref:../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Upgrading channels and release paths]
+* xref:../updating/understanding-upgrade-channels-release.adoc#upgrade-version-paths_understanding-upgrade-channels-releases[Upgrading version paths]
+* xref:../updating/understanding-upgrade-channels-release.adoc#fast-stable-channel-strategies_understanding-upgrade-channels-releases[Understanding fast and stable channel use and strategies]
+* xref:../updating/understanding-upgrade-channels-release.adoc#restricted-network-clusters_understanding-upgrade-channels-releases[Understanding restricted network clusters]
+* xref:../updating/understanding-upgrade-channels-release.adoc#switching-between-channels_understanding-upgrade-channels-releases[Switching between channels]
+
+[id="updating-clusters-overview-update-cluster-using-web-console"]
+== Updating a cluster using the web console
+xref:../updating/updating-cluster-within-minor.adoc#updating-cluster-within-minor[Updating a cluster within a minor version using the web console]: You can update an {product-title} cluster by using the web console. You can the update a cluster between minor versions. For more information, see the following items:
```

Comment:
Addressed

Reply:
I noticed that for some reason the [web console update page](https://deploy-preview-45298--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-cluster-within-minor.html) in the Upgrading book states that the procedures are for upgrading within a minor release, but that you can use the same instructions to update between minor versions.

However, the [CLI update page](https://deploy-preview-45298--osdocs.netlify.app/openshift-enterprise/latest/updating/updating-cluster-cli.html#updating-cluster-cli) only references updates within a minor version.

You could change "Updating a cluster within a minor version using the web console" to "Updating a cluster using the web console", to mirror the target heading and to accommodate the point that the instructions can be for between minor releases too, in this case. You could also remove "within a minor version" from the text on this line.

Reply:
Beyond this PR, it might be useful to review how to make the web console and CLI instructions consistent by providing details about how to update between minor releases using the CLI. I thought I'd add that note here, although it is likely another work item out of scope for this review.

Reply:
Addressed

Noted the point 2